test/unit/Page.test.js

```diff
@@ -0,0 +1,15 @@
+const Page = require('../../lib/Page');
+
+test('Page#collectIncludedFiles collects included files from 1 dependency object', () => {
+  const page = new Page({});
+  page.collectIncludedFiles([{ to: 'somewhere' }]);
+
+  expect(page.includedFiles).toEqual({ somewhere: true });
+});
+
+test('Page#collectIncludedFiles ignores other keys in dependency', () => {
```

Comment:
Perhaps in addition to these tests, a test for `Page#collectIncludedFiles` to not include anything from a dependency object could be made as well?